source/documentation/getting_started/quick_start.md

```diff
@@ -16,6 +16,8 @@ You should include your key in all requests using the `Authorization` header:
 
 Find the base URL for each register on its API inspector. For example, the API inspector for the `local-authority-eng` register is located at `https://local-authority-eng.register.gov.uk/`. 
 
+The base URL of a register is always of the form `https://{register}.{domain}`. For example, the base URL for the `local-authority-eng` register is `https://local-authority-eng.register.gov.uk`, so the domain for this register is `register.gov.uk`. 
```

Comment:
Rather than use the `datatypes.md` file to have this content as part of a tutorial, I've suggested expanding the content about base URLs in the 'Quick start guide' section. This may not be the best way, so I am open to alternative suggestions: this avoids the problem of having two 'For example' structures in quick succession in `datatypes.md` (i.e. example about what a domain is, then example about how to find out what the datatype is).